src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -680,55 +698,39 @@ export default {
 		// Checks whether the conversations list is scrolled all the way to the top
 		// or not
 		handleScroll() {
-			this.isScrolledToTop = this.$refs.container.scrollTop === 0
-		},
-		elementIsAboveViewpoint(container, element) {
-			return element.offsetTop < container.scrollTop
-		},
-		elementIsBelowViewpoint(container, element) {
-			return element.offsetTop + element.offsetHeight > container.scrollTop + container.clientHeight
+			this.isScrolledToTop = this.$refs.scroller.$el.scrollTop === 0
 		},
+
+		/**
+		 * Find position of unread conversation below viewport
+		 */
 		handleUnreadMention() {
-			this.unreadNum = 0
-			const unreadMentions = document.querySelectorAll('.unread-mention-conversation')
-			unreadMentions.forEach(x => {
-				if (this.elementIsBelowViewpoint(this.$refs.container, x)) {
-					if (this.unreadNum === 0) {
-						this.firstUnreadPos = x.offsetTop
+			this.$nextTick(() => {
+				this.unreadNum = 0
```

Comment:
As we don't use an exact amount of unread mentions, maybe, transform it to the boolean value? Or discard it and keep only `firstUnreadIndex` for both tracking and navigating purposes?